Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -51,7 +51,9 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
         this._highlightedAttributes = new Set;
         this._recentlyModifiedAttributes = new Map;
         this._closeTagTreeElement = null;
-        this._layoutBadgeElement = null;
+
+        this._showBadges = !!showBadges;
+        this._elementForBadge = new Map;
```

Comment:
The singular variable name makes me think this is the element in which the badge is placed, not that it is a map of badge type to element. Maybe `_badgeElementsByType` or something similar?